finality-aleph/src/validator_network/service.rs

```diff
@@ -206,7 +206,7 @@ impl<D: Data, A: Data, ND: Dialer<A>, NL: Listener> Service<D, A, ND, NL> {
                 },
                 // periodically reporting what we are trying to do
                 _ = status_ticker.tick() => {
-                    info!(target: "validator-network", "Manager status report: {}.", self.manager.status_report())
+                    info!(target: "validator-network", "Validator Network Manager status report: {}", self.manager.status_report())
```

Comment:
Let's try to cut on unnecessary words here:
"Validator Network Manager status: ... "

Is there any other log that is not from the manager? If not, you can also omit "Manager"

Reply:
If we ensure the target is always displayed we can even omit the "Validator Network", leaving just "Status" (half-joking – I'm not sure what triggers this, but targets are often not logged).

Reply:
There is no other status log from Validator Network. I will remove "Manager"